.buildkite/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+mkdir build_src
+
+if [[ $LE_TRIGGERED_FROM_BUILD_ID ]]
+then
+  echo "--- Downloading from triggered build"
+  buildkite-agent artifact download "dist/*.tar.gz" . --build $LE_TRIGGERED_FROM_BUILD_ID
+  mv dist build_src
+  if [[ -z $BUILDKITE_TRIGGERED_FROM_BUILD_ID ]]
```

Comment:
```suggestion
  # If $BUILDKITE_TRIGGERED_FROM_BUILD_ID is zero-length, no build was triggered
  if [[ -z $BUILDKITE_TRIGGERED_FROM_BUILD_ID ]]
```

Reply:
This script is pretty much only to ever run using Buildkite, and that's a buildkite functionality. That var will only ever be an empty string or a populated string, and I don't think anyone who doesn't know what they're doing should be editing this file. 

Not sure this comment is worth it?

Reply:
> I don't think anyone who doesn't know what they're doing should be editing this file

Well, it's easier to know what one is doing if the code has helpful comments :)

Doesn't matter.. if it's likely to change soon, it's not worth spending time on, I guess.